app/_src/kubernetes-ingress-controller/upgrade/kic.md

```diff
@@ -73,6 +73,26 @@ complete the following steps to upgrade to version v1.1 of the CRD:
 
 {% endif_version %}
 
+{% if_version gte:3.4.x %}
+
+### Update Gateway API from v1.1 to v1.2
+
+Starting from version 3.4, {{ site.kic_product_name }} supports Gateway API version 1.2.
+There is a breaking change in gateway API 1.2 to remove the `v1alpha2` version of `GRPCRoute` and `ReferenceGrant`.
```

Comment:
Maybe we could suggest something like:

```
kubectl get GRPCRoute,ReferenceGrant --all-namespaces -o jsonpath='{.items[*].apiVersion}' | tr ' ' '\n' | sort | uniq -c
```

And show an invalid output?

Reply:
Added a script to check existing resources.